modules/ossm-rn-new-features-1x.adoc

```diff
@@ -40,6 +40,121 @@ Result – If changed, describe the current user experience
+
+Your cluster is impacted by this vulnerability if you have authorization policies using `ALLOW action + notPaths` field or `DENY action + paths field` patterns. These patterns are vulnerable to unexpected policy bypasses
+
+Your cluster is NOT impacted by this vulnerability if:
+
+* You don’t have authorization policies
+* Your authorization policies don’t define `paths` or `notPaths` fields.
+* Your authorization policies use `ALLOW action + paths` field or `DENY action + notPaths` field patterns. These patterns could only cause unexpected rejection instead of policy bypasses. The upgrade is optional for these cases.
+
+[NOTE]
+====
+The {ProductName} configuration location for path normalization is different from the Istio configuration.
+====
+
+=== Updating the path normalization configuration
+
+Istio authorization policies can be based on the URL paths in the HTTP request.
+https://en.wikipedia.org/wiki/URI_normalization[Path normalization], also known as URI normalization, modifies and standardizes the incoming requests' paths, so that the normalized paths can be processed in a standard way.
+Syntactically different paths may be equivalent after path normalization.
+
+Istio supports the following normalization schemes on the request paths, before evaluating against the authorization policies and routing the requests:
+
+.Normalization schemes
+[options="header"]
+[cols="a, a, a, a"]
+|====
+| Option | Description | Example |Notes
+|`NONE`
+|No normalization is done. Anything received by Envoy will be forwarded exactly as-is to any backend service.
+|`../%2Fa../b` is evaluated by the authorization policies and sent to your service.
+|This setting is vulnerable to CVE-2021-31920.
+
+|`BASE`
+|This is currently the option used in the *default* installation of Istio. This applies the https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto#envoy-v3-api-field-extensions-filters-network-http-connection-manager-v3-httpconnectionmanager-normalize-path[`normalize_path`] option on Envoy proxies, which follows https://tools.ietf.org/html/rfc3986[RFC 3986] with extra normalization to convert backslashes to forward slashes.
+|`/a/../b` is normalized to `/b`. `\da` is normalized to `/da`.
+|This setting is vulnerable to CVE-2021-31920.
+
+| `MERGE_SLASHES`
+| Slashes are merged after the _BASE_ normalization.
+| `/a//b` is normalized to `/a/b`.
+|Update to this setting to mitigate CVE-2021-31920.
+
+|`DECODE_AND_MERGE_SLASHES`
+|The most strict setting when you allow all traffic by default. This setting is recommended, with the caveat that you will need to thoroughly test your authorization policies routes. https://tools.ietf.org/html/rfc3986#section-2.1[Percent-encoded] slash and backslash characters (`%2F`, `%2f`, `%5C` and `%5c`) are decoded to `/` or `\`, before the `MERGE_SLASHES` normalization.
```

Comment:
```suggestion
|The strictest setting when you allow all traffic by default. Thoroughly test your authorization policies routes. https://tools.ietf.org/html/rfc3986#section-2.1[Percent-encoded] slash and backslash characters (`%2F`, `%2f`, `%5C` and `%5c`) are decoded to `/` or `\`, before the `MERGE_SLASHES` normalization.
```
Check for duplicate issue.
IBM SG: "Product security: Avoid making general statements that a product or service is "secure" when you discuss the security capabilities of a product or service. Stating that a product is “secure” or provides
"secure functionality" could expose IBM to liability. You can use phrases such as “security enhanced”
or "in a security-rich environment." In most cases, avoid "recommending" that users do something,
especially if the recommendation is vague or unexplained. Instead, describe why the action is
problematic or beneficial."

Reply:
The whole point of this PR is to recommend settings to mitigate the CVE.

Reply:
How about this, `To mitigate CVE-<blah> and similar security issues, use this setting. Thoroughly test...`